README.md

```diff
@@ -34,25 +34,26 @@ The package also supports 2 additional build modes:
 This project contains a mix of natively defined classes and classes generated by the Protocol
 Buffer Compiler from definitions in the `proto/` directory. Copies of the generated classes  are
 included in the `src/` directory to allow the build process to succeed without a dependency on
-the Protocol Buffer compiler, `protoc`, being in the system PATH. 
+the Protocol Buffer compiler, `protoc`, being in the system PATH.
 
-If a file in `proto/` is modified or you need to manually re-generate the protos, you'll need to
-add `protoc` to your system's PATH and generate new compiled protos by running:
+If a file in `proto/` is modified or you need to manually re-generate the protos, you'll need to:
+
+* Add `protoc` ([download](https://github.com/protocolbuffers/protobuf/releases)) to your system PATH
+* Add `protoc-gen-grpc-web` ([download](https://github.com/grpc/grpc-web/releases)) to your system PATH
```

Comment:
Can you mention which version you used last time? So we can avoid version diff causing hard to troubleshoot problems

Reply:
I added a comment at the bottom of the section with the protoc and protoc-gen-grpc-web versions.